src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
```

Comment:
it's better to use `span`
```suggestion
        <span> Posted by </span>
```